src/schemas/AlertThresholdBaseObject.yaml

```diff
@@ -1,6 +1,4 @@
 type: object
-required:
-  - value
```

Comment:
This Base object does not define `value`. Both objects using this Base already require `value`.